templates/eap-cd-amq-persistent-s2i.json

```diff
@@ -807,7 +807,7 @@
                                 },
                                 "ports": [
                                     {
-                                        "name": "jolokia",
+                                        "name": "console-jolokia",
```

Comment:
@maschmid @tremes FYI, updated the AMQ broker jolokia to match what the released AMQ templates have.